packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -618,6 +618,101 @@ Expected response body:
 }
 ```
 
+### Client_Structure_MultiClient
+
+- Endpoints:
+  - `get /one`
+  - `get /three`
+  - `get /three`
+  - `get /one`
+  - `get /three`
+  - `get /three`
+
+Include multiple clients in the same spec.
+
+```ts
+const clientA = new ClientAClient("multi-client");
+const clientB = new ClientBClient("multi-client");
+
+clientA.renamedOne();
+clientA.renamedThree();
+clientA.renamedFive();
+
+clientB.renamedTwo();
+clientB.renamedFour();
+clientB.renamedSix();
+```
+
+### Client_Structure_RenamedOperation
+
+- Endpoints:
+  - `get /one`
+  - `get /three`
+  - `get /three`
+  - `get /one`
+  - `get /three`
+  - `get /three`
+
+This is to show we can have more than one operation group in a client. The client side should be able to call the api like
+
+```ts
+const client = new RenamedOperationClient("renamed-operation");
+
+client.renamedOne();
+client.renamedThree();
+client.renamedFive();
+
+client.group.renamedTwo();
+client.group.renamedFour();
+client.group.renamedSix();
+```
+
+### Client_Structure_Service
+
+- Endpoints:
+  - `get /three`
+  - `get /four`
+  - `get /five`
+  - `get /six`
+  - `get /one`
+  - `get /two`
+
+This is to show that if we don't do any customization. The client side should be able to call the api like
+
+```ts
+const client = new MultiClient("default");
+client.one();
+client.two();
+client.three();
+client.four();
+client.five();
+client.six();
+```
+
+### Client_Structure_TwoOperationGroup
```

Comment:
@timotheeguerin Let us know if the test case hierarchy (e.g. `Client_Structure_TwoOperationGroup`) is good for you.

I am a bit unsure about "Structure", but don't have good recommendation.